service/controller/v15/resource/cloudformation/current.go

```diff
@@ -33,15 +32,17 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	// stack. We dispatch our custom StackState structure and enrich it with all
 	// information necessary to reconcile the cloudformation resource.
 	var stackOutputs []*cloudformation.Output
-	var stackStatus string
 	{
-		stackOutputs, stackStatus, err = sc.CloudFormation.DescribeOutputsAndStatus(stackName)
+		stackOutputs, err = sc.CloudFormation.DescribeOutputsAndStatus(stackName)
 		if cloudformationservice.IsStackNotFound(err) {
 			r.logger.LogCtx(ctx, "level", "debug", "message", "did not find the guest cluster main stack in the AWS API")
 			return StackState{}, nil
 
-		} else if cloudformationservice.IsOutputsNotAccessible(err) {
+		} else if cloudformationservice.IsStackInTransition(err) {
 			r.logger.LogCtx(ctx, "level", "debug", "message", "the guest cluster main stack output values are not accessible due to stack state transition")
```

Comment:
Can we distinguish fact and reason logs?

Reply:
It was all mixed. Addressed here: https://github.com/giantswarm/aws-operator/pull/1099/commits/31680f4bab739984d5c1660fc679f36670ebbf5a